topics/assembly/tutorials/flye-assembly/workflows/main_workflow.ga

```diff
@@ -0,0 +1 @@
gmented\": \"false\", \"fragmented_max_indent\": \"50\", \"upper_bound_assembly\": \"false\", \"upper_bound_min_con\": \"2\"}, \"assembly\": {\"type\": \"genome\", \"__current_case__\": 0, \"ref\": {\"use_ref\": \"false\", \"__current_case__\": 1, \"est_ref_size\": null}, \"orga_type\": \"\"}, \"circos\": \"false\", \"contig_thresholds\": \"0,1000\", \"extensive_mis_size\": \"1000\", \"genes\": {\"gene_finding\": {\"tool\": \"none\", \"__current_case__\": 0}, \"rna_finding\": \"false\", \"conserved_genes_finding\": \"false\"}, \"in\": {\"custom\": \"false\", \"__current_case__\": 1, \"inputs\": {\"__class__\": \"ConnectedValue\"}}, \"k_mer\": {\"k_mer_stats\": \"\", \"__current_case__\": 1}, \"large\": \"false\", \"min_contig\": \"500\", \"scaffold_gap_max_size\": \"1000\", \"skip_unaligned_mis_contigs\": \"true\", \"split_scaffolds\": \"false\", \"strict_NA\": \"false\", \"unaligned_part_size\": \"500\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "5.0.2+galaxy1", "type": "tool", "uuid": "1b460c75-f557-44d5-b837-ecb93894d077", "workflow_outputs": [{"label": "Quast on input dataset(s):  PDF report", "output_name": "report_pdf", "uuid": "7623ccb4-6173-4fdb-b92b-1d99c60e4bb1"}, {"label": "Quast on input dataset(s):  HTML report", "output_name": "report_html", "uuid": "27a258b6-c1af-4aba-8ee6-263aa9caf519"}, {"label": "Quast on input dataset(s): Log", "output_name": "log", "uuid": "a2ea73dc-b61c-4904-a961-bd88563a9e40"}, {"label": "Quast on input dataset(s): tabular report", "output_name": "quast_tabular", "uuid": "0be21622-4677-4c77-a90e-f65ea42b124f"}]}, "4": {"annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/busco/busco/5.2.2+galaxy0", "errors": null, "id": 4, "input_connections": {"input": {"id": 1, "output_name": "consensus"}}, "inputs": [], "label": null, "name": "Busco", "outputs": [{"name": "busco_table", "type": "tabular"}], "position": {"bottom": 907.5, "height": 134, "left": 1072.5, "right": 1272.5, "top": 773.5, "width": 200, "x": 1072.5, "y": 773.5}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/busco/busco/5.2.2+galaxy0", "tool_shed_repository": {"changeset_revision": "0d243f458b53", "name": "busco", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"adv\": {\"evalue\": \"0.001\", \"limit\": \"3\", \"auto_lineage\": null, \"outputs\": null}, \"busco_mode\": {\"mode\": \"geno\", \"__current_case__\": 0, \"use_augustus\": {\"use_augustus_selector\": \"no\", \"__current_case__\": 0}}, \"input\": {\"__class__\": \"ConnectedValue\"}, \"lineage_dataset\": \"acidobacteria_odb10\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "5.2.2+galaxy0", "type": "tool", "uuid": "7ef8e447-ff0a-4564-9199-7ee322cb8f2f", "workflow_outputs": [{"label": "Busco on input dataset(s): full table", "output_name": "busco_table", "uuid": "bd413388-fcf7-4b8d-9e65-ee8ad232abb0"}]}}, "tags": [], "uuid": "2a699ca5-035b-4c84-b7f0-9c3103c035a3", "version": 8}
```

Comment:
Can you update the tools and reexport the workflow?

Reply:
done